Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>

The dirs project says that the changes between 3 and 4 are:

==========
* **BREAKING CHANGE** The behavior of `executable_dir` has been adjusted to not depend on `$XDG_DATA_HOME`.
  Code, which assumed that setting the `$XDG_DATA_HOME` environment variable also impacted `executable_dir` if
  the `$XDG_BIN_HOME` environment variable was not set, requires adjustment.
* Add support for `XDG_STATE_HOME`.
==========

Neither of these are relevant for sequoia-ipc, so we can relax the upstream dependency a bit.

If we update dirs in debian to 4, it should be as simple as dropping this patch.


--- a/Cargo.toml
+++ b/Cargo.toml
@@ -35,7 +35,7 @@
 version = "0.14"
 
 [dependencies.dirs]
-version = "4.0"
+version = "3"
 
 [dependencies.fs2]
 version = "0.4.2"
